packages/design-system/src/icons/iconNames.ts

```
@@ -1,3 +1,28 @@
+  'ic_arrow_down',
+  'ic_arrow_right',
+  'ic_error',
+  'main_logo',
+  'plus_icon',
+  'popup_logo',
+  'white_order',
+] as const;
+export type IconName = (typeof iconNames)[number];
+export const iconNames = [
+  'close-button',
+  'default_order',
```

Biome (1.9.4)

[error] Shouldn't redeclare 'iconNames'. Consider to delete it or rename it.

'iconNames' is defined here:


(lint/suspicious/noRedeclare)